languages/python/sqlalchemy-oso/tests/test_roles.py

```diff
@@ -483,9 +484,8 @@ def test_duplicate_role_name_same_resource(init_oso):
             }
         };
         """
-    oso.load_str(policy)
-    with pytest.raises(OsoError):
-        oso.roles.synchronize_data()
+    with pytest.raises(ParserError):
+        oso.load_str(policy)
```

Comment:
FYI @saolsen we don't need to check for duplicate keys in the same `resource/4` any more, but maybe we should still check for two `resource/4` predicates defined for the same type and the same role name?